modules/soe_paragraph_stories/config/install/auto_entitylabel.settings.paragraph.soe_stories.yml

```diff
@@ -0,0 +1,6 @@
+status: 0
```

Comment:
auto_entitylabel can be removed from this module.

Reply:
I don't think it is needed unless you know something I dont.

Reply:
removed.